tsify-next-macros/src/typescript/ts_type_from_name.rs

```diff
@@ -117,7 +117,7 @@ impl TsType {
                 nanos_since_epoch: Self::NUMBER;
             },
 
-            "Range" | "RangeInclusive" => {
+            "Range" | "RangeInclusive" if args.len() == 1 => {
```

Comment:
Can you add a comment here explaining why we do the args check?

Reply:
https://github.com/siefkenj/tsify/pull/45/commits/d9c0a866d8ec4831dabe9a5ab326738437b675ff
Comment added.